Makefile

```diff
@@ -60,3 +60,6 @@ docker/%:
 	$(DOCKER_COMPOSE) run --rm app make $*
 
 build: clean venv tox
+
+changelog:
+	gitchangelog
```

Comment:
do you mean `venv/bin/changelog`?

Reply:
The command is `venv/bin/gitchangelog`. I'll adjust it. Also the dependency.

Reply:
Fixed.